tests/unit/test_completer.py

```diff
@@ -73,7 +74,9 @@
           '--no-verify-ssl', '--no-paginate', '--no-sign-request', '--output',
           '--profile', '--starting-token', '--max-items', '--page-size',
           '--region', '--version', '--color', '--query', '--ca-bundle',
-          '--generate-cli-skeleton', '--cli-input-json', '--cli-read-timeout'])),
+          '--generate-cli-skeleton', '--cli-input-json', '--cli-read-timeout',
```

Comment:
We really need to fix this.  We shouldn't have to update the completer whenever we add new command line options.